src/app/Characters/services/CharacterService.js

```diff
@@ -0,0 +1,12 @@
+import httpClient from '../../AxiosHttpClient';
+
+const CHARACTERS_ROUTE = 'characters';
+
+export default {
+  async fetchCharacters() {
+    const GET_ALL_CHARACTERS_ROUTE = `${CHARACTERS_ROUTE}/getAll`;
```

Comment:
You only need to use one of these two words. You need to decide between:
- *Fetch*
- *Get*